pkgs/applications/misc/far2l/default.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv, fetchgit, fetchFromGitHub, makeWrapper, cmake, pkgconfig, wxGTK30, glib, pcre, m4, git, bash,
+  xdg_utils, xterm, gvfs, zip, unzip, gzip, bzip2, gnutar, p7zip, xz }:
+
+stdenv.mkDerivation rec {
+  name = "far2l-2.1";
+  rev = "c2f2b89db31b1c3cb9bed53267873f4cd7bc996d";
+
+  src = fetchgit {
+    url = "https://github.com/elfmz/far2l";
+    rev = rev;
+    leaveDotGit = true;
```

Comment:
yes, git is used by the build scripts to put last commit date and some hash digits into version string.
not a big deal, but removing ```leaveDotGit``` would require adding patch of makefiles (at least to replace ```git ...```  with ```echo 2017-03-17-${rev}```)